### PR TITLE
added how to delete meshObject guide

### PR DESCRIPTION
**Pull request by younGihan**

### CODE REVIEW

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+id: meshstack.how-to.delete-meshObjects
+title: How to delete meshObjects in the meshPanel
+---
+
+Part of the cloud lifecyle inside meshStack is the deletion of the meshCustomers, meshProject and meshTenants.
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:6]:
```suggestion
Part of the cloud lifecyle inside meshStack is the deletion of meshCustomers, meshProject and meshTenants.
```

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The following guide provides insight for users and admins how to proceed the deletion of their cloud accounts and meshObjects via the meshPanel
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:7]:
```suggestion
The following guide provides a guide for users how to proceed with the deletion of their cloud environments and workspaces via the meshPanel
```

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Step-by-Step Guide
+
+## Delete meshTenant from my meshProject
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:11]: please add what rights are necessary to delete the tenant

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+If you want to delete just a meshTenant e.g. AWS Account without deleting your complete meshProject you can use the meshTenant deletion.
+
+The following step-by-step guide shows how to successfully request the deletion of an meshTenant:
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Go to your meshProject in which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
+3. Click on "View more details" for the meshTenant you want to delete !["View more details"](./assets/project/project-nav-to-tenant.png)
+4. Click on the "Deletion" tab and follow the deletion instusction for the meshTenant !["Delete meshTenant"](./assets/project/project-tenant-deletion.png)
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:20]: please put the tenant deletion approval below it and make transparent that this needs to be done by an admin

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Delete meshProjects
+
+The deletion of an meshProjects is quite similar to the meshTenant deletion.
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:24]:
```suggestion

```
not important

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+The following step-by-step guide shows how to successfully request a meshProject deletion:
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Go to your meshProject in which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:28]:
```suggestion
2. Go to your meshProject from which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
```

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+3. Click on the "Deletion" tab and follow the deletion instusction for the meshProject !["Delete meshTenant"](./assets/project/project-deletion.png)
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:29]:
```suggestion
3. Click on the "Deletion" tab and follow the deletion instruction for the meshProject !["Delete meshTenant"](./assets/project/project-deletion.png)
```

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+## Delete meshCustomer
+
+Your user needs the "Customer Owner" role to proceed with the deletion of the meshCustomer.
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Click on the "Deletion" tab and follow the deletion instusction for the meshCustomer !["Delete meshTenant"](./assets/customer/customer-deletion.png)
```
Comment (chewbeccy) [inline docs/meshstack.how-to.delete-meshObjects.md:36]:
```suggestion
2. Click on the "Deletion" tab and follow the deletion instructions for the meshCustomer !["Delete meshTenant"](./assets/customer/customer-deletion.png)
```

**Review by chewbeccy — APPROVED** [review]
There are small things missing and some typos I suggested fixes to. Thanks for creating the documentation

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+id: meshstack.how-to.delete-meshObjects
+title: How to delete meshObjects in the meshPanel
+---
+
+Part of the cloud lifecyle inside meshStack is the deletion of the meshCustomers, meshProject and meshTenants.
+The following guide provides insight for users and admins how to proceed the deletion of their cloud accounts and meshObjects via the meshPanel
+
+## Step-by-Step Guide
+
```
Comment (felixzieger) [inline docs/meshstack.how-to.delete-meshObjects.md:10]:
```suggestion
```

or plural (Step-by-Step Guides) and indentation of the invidual guides one level below (so three hashtags: ### Delete meshTenant from my meshProject)

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+If you want to delete just a meshTenant e.g. AWS Account without deleting your complete meshProject you can use the meshTenant deletion.
+
+The following step-by-step guide shows how to successfully request the deletion of an meshTenant:
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Go to your meshProject in which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
+3. Click on "View more details" for the meshTenant you want to delete !["View more details"](./assets/project/project-nav-to-tenant.png)
+4. Click on the "Deletion" tab and follow the deletion instusction for the meshTenant !["Delete meshTenant"](./assets/project/project-tenant-deletion.png)
+
+## Delete meshProjects
```
Comment (felixzieger) [inline docs/meshstack.how-to.delete-meshObjects.md:22]:
```suggestion
## How to delete a meshProject
```

File: docs/meshstack.how-to.delete-meshObjects.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+The deletion of an meshProjects is quite similar to the meshTenant deletion.
+The following step-by-step guide shows how to successfully request a meshProject deletion:
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Go to your meshProject in which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
+3. Click on the "Deletion" tab and follow the deletion instusction for the meshProject !["Delete meshTenant"](./assets/project/project-deletion.png)
+
+## Delete meshCustomer
```
Comment (felixzieger) [inline docs/meshstack.how-to.delete-meshObjects.md:31]:
```suggestion
## How to delete a meshCustomer
```

File: docs/meshstack.how-to.delete-meshObjects.md (hunk 3 above)
Comment (felixzieger) [inline docs/meshstack.how-to.delete-meshObjects.md:11]:
```suggestion
## How to delete a meshTenant
```

Comment (younGihan) [reply docs/meshstack.how-to.delete-meshObjects.md]: added into each section which role is required

File: docs/meshstack.how-to.delete-meshObjects.md (hunk 4 above)
Comment (younGihan) [reply docs/meshstack.how-to.delete-meshObjects.md:20]: done

File: docs/meshstack.how-to.delete-meshObjects.md (hunk 9 above)
Comment (younGihan) [reply docs/meshstack.how-to.delete-meshObjects.md:10]: deleted the headline

**Review by chewbeccy — APPROVED** [review]

**Review by felixzieger — APPROVED** [review]

**Pull request merged**